loops.js

```diff
@@ -87,5 +109,19 @@
 // etc.
 // ==========================================
 
-
+for(let i = 1; i <= 100; i++) {
+    if(i%3===0 && i%5===0) {
+        console.log("FizzBuzz");
+    }
+
```

Comment:
Het gebruik van if-else om cijfers te categoriseren met verschillende prefixes is correct geïmplementeerd. De aanpak is logisch en de code is makkelijk te volgen.

Reply:
Overweeg het gebruik van else if in plaats van afzonderlijke if-statements na de eerste controle om de efficiëntie iets te verbeteren.